classes/install/form/InstallForm.inc.php

```diff
@@ -43,7 +43,7 @@ class InstallForm extends MaintenanceForm
     public $supportedDatabaseDrivers = [
         // <driver> => array(<php-module>, <name>)
         'mysqli' => ['mysqli', 'MySQLi'],
-        'postgres9' => ['pgsql', 'PostgreSQL'],
+        'postgres' => ['pgsql', 'PostgreSQL'],
```

Comment:
I'll double check. I wrote this `postgres9`, and at first I thought it was just a typo 😁

Reply:
I've just checked and it makes sense to keep the postgres9 :)

Should I create an issue to replace these ADOdb identifiers by Laravel? Also, the `mysql` driver can be removed, the extension has been gone since PHP 7.0

Reply:
This is a really good reason to avoid wandering out of scope when making changes -- it's a very subtle change unrelated to the issue that could've gone unnoticed and been very tricky to debug!

We'll remove the ADODB driver identifiers and generally clean up database connection stuff when we're able to remove ADODB fully (after the upgrade to 3.3.x gets removed). But yes, feel free to remove the `mysql` driver from the installation form.